build/release/verify.js

```diff
@@ -22,7 +19,15 @@ const SRC_REPO = "https://github.com/jquery/jquery.git";
 const CDN_URL = "https://code.jquery.com";
 const REGISTRY_URL = "https://registry.npmjs.org/jquery";
 
-const rstable = /^(\d+\.\d+\.\d+)$/;
+const excludeFromCDN = [
+	"jquery.factory.js",
+	"jquery.factory.slim.js",
+	"jquery.factory.module.js",
+	"jquery.factory.slim.module.js",
```

Comment:
Since the logic is to basically exclude `package.json` & factory files, it would be safer to exclude the pattern `jquery.factory.*` or regex `/^jquery\.factory\./` and match file names against it.